### PR TITLE
Release v1.0.1

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

**Review by MrAlias — APPROVED** [review]

**Review by jkwatson — APPROVED** [review]

**Review by tigrannajaryan — APPROVED** [review]
Approving. This fixes an editorial mistake in 1.0.0, so I think qualifies for a patch number increase.
@bogdandrutu do we need a cooldown period before merging this?

**Comment by bogdandrutu** [comment]
@tigrannajaryan good question, I just want to make sure that languages will not take the wrong value and release with that. I think we can do patch releases like this often. No idea, but would wait for 4 TC members anyway @open-telemetry/technical-committee please approve :)

**Review by reyang — APPROVED** [review]
LGTM.

**Review by cijothomas — APPROVED** [review]

**Review by jmacd — APPROVED** [review]

**Review by iNikem — APPROVED** [review]

**Review by arminru — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ release.
 
 ## Unreleased
 
+## v1.0.1 (02-10-2021)
```
Comment (Oberon00) [inline CHANGELOG.md:10]:
```suggestion
## v1.0.1 (02-11-2021)
```

**Review by carlosalberto — APPROVED** [review]

**Pull request merged**